garage/tf/algos/dqn.py

```diff
@@ -17,37 +9,67 @@
 
 
 class DQN(OffPolicyRLAlgorithm):
-    """
-    DQN.
-
-    Args:
-        env_spec (garage.envs.env_spec.EnvSpec): Environment specification.
-        max_path_length (int): Maximum path length. The episode will terminate
-            when length of trajectory reaches max_path_length.
-        qf_lr (float): Learning rate for Q-Function.
-        qf_optimizer (tf.Optimizer): Optimizer for Q-Function.
-        discount (float): Discount factor for rewards.
-        name (str): Name of the algorithm.
-        target_network_update_freq (int): Frequency of updating target network.
-        grad_norm_clipping (float): Maximum clipping value for clipping tensor
-            values to a maximum L2-norm. It must be larger than 0. If None,
-            no gradient clipping is done. For detail, see docstring for
-            tf.clip_by_norm.
-        double_q (bool): Bool for using double q-network.
+    """DQN from https://arxiv.org/pdf/1312.5602.pdf.
```

Comment:
the original position of this documentation was fine. please just make sure all the algorithms are consistent with where they put the documentation (either in module or class).